src/pages/ProfilePage.js

```diff
@@ -231,6 +237,15 @@ function ProfilePage(props) {
                             ) : null}
                             {shouldShowLocalTime && <AutoUpdateTime timezone={timezone} />}
                         </View>
+                        {chatReportWithCurrentUser.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN && (
+                            <MenuItemWithTopDescription
```

Comment:
We can move this to their own component with passed `reportID` from `chatReportWithCurrentUser`. Inside the component, we can get the report from Onyx, so it can be updated accordingly.


https://github.com/Expensify/App/assets/25520267/bd8c6b20-0840-4753-b15b-99773efc6fbc

Reply:
> We can move this to their own component with passed reportID from chatReportWithCurrentUser. Inside the component, we can get the report from Onyx, so it can be updated accordingly.

Yeah, good point, I have used the same concept but on the same page. Let me know if you have any concerns.